drivers/telescope/lx200telescope.cpp

```diff
@@ -120,9 +121,10 @@ bool LX200Telescope::initProperties()
 
     char siteName[64] = {"NA"};
     IUGetConfigText(getDeviceName(), "Site Name", "Name", siteName, 64);
-    IUFillText(&SiteNameT[0], "Name", "Name", siteName);
+    IUFillText(&SiteNameT[0], "SiteName", "SiteName", siteName);
```

Comment:
I don't think we need to change this. It would break users existing config files. It also doesn't really add any benefit, "Name" works OK.